pkgs/applications/misc/osm2xmap/default.nix

```diff
@@ -0,0 +1,38 @@
+{ stdenv, fetchFromGitHub, libroxml, proj, libyamlcpp, boost } :
+
+stdenv.mkDerivation rec {
+  name = "osm2xmap-${version}";
+  version = "2.0";
+
+  src = fetchFromGitHub {
+    sha256 = "1d3f18wzk240yp0q8i2vskhcfj5ar61s4hw83vgps0wr2aglph3w";
+    repo = "osm2xmap";
+    owner = "sembruk";
+    rev = "v${version}";
+  };
+
+  makeFlags = [
+    "GIT_VERSION=$(version)"
+    "GIT_TIMESTAMP="
+    "SHAREDIR=$(out)/share"
+    "INSTALL_BINDIR=$(out)/bin"
+    "INSTALL_BINDIR=$(out)/share/man/man1"
+    "INSTALL_SHAREDIR=$(out)/share"
```

Comment:
I cleaned things up a bit here.

Reply:
This broke the build. `osm2xmap` is not available anymore after these changes.

Reply:
Typo: I wrote `INSTALL_BINDIR=$(out)/share/man/man1` instead of `INSTALL_MANDIR=$(out)/share/man/man1`